bokeh/core/property/dataspec.py

```diff
@@ -482,6 +484,8 @@ def prepare_value(self, cls, name, value):
 
 class NullDistanceSpec(DistanceSpec):
```

Comment:
@bryevdv, at this point, I'm not sure what to expect from this property type. Does it support `null` or `{value: null}` or both?